concept/ConceptManager.java

```diff
@@ -81,7 +81,7 @@ public EntityType putEntityType(String label) {
     @Nullable
     @CheckReturnValue
     public EntityType getEntityType(String label) {
-        final Type type = getType(label);
+        final Type type = getThingType(label);
```

Comment:
```suggestion
        final ThingType type = getThingType(label);
```
Although the difference is academic right now, we are planning to reintroduce `asThingType`, `asEntityType` etc. to `Type` class which will make stricter typing more relevant.

Reply:
Also fixed in https://github.com/graknlabs/client-java/commit/ace3adaff902dab2bca5d1d1961d29e164c4ddf8